k2/csrc/ragged_ops.cu

```diff
@@ -1010,10 +997,11 @@ static Array1<int32_t> GetTransposeReorderingThreeAxesCuda(Ragged<int32_t> &src,
 
   int32_t n = src.values.Dim();
   Array1<int32_t> ans = Range(context, n, 0);
-  K2_CUDA_SAFE_CALL(mgpu::segmented_sort(ans.Data(),          // keys
-                                         ans.Dim(),           // count
-                                         segments.Data(),     // segments
-                                         segments.Dim() - 1,  // num_segments
```

Comment:
@csukuangfj not sure why you `-1` here, but all tests get passed by moving this.

Reply:
I tried both of them and both worked. I don't remember why I chose `-1`.